packages/components/src/components/Autocomplete/Autocomplete.tsx

```diff
@@ -0,0 +1,206 @@
+                    handleToggleOptionPress(option);
+                }}
+            />
+        );
+    };
+
+    return (
+        <View
+            onLayout={event => {
+                const layout = event.nativeEvent.layout;
+                setInputLayout(layout);
+            }}
+        >
+            <Input
+                ref={inputRef}
+                {...restProps}
+                value={`${
+                    selectedToggleOptions.length ? selectedToggleOptions.join(", ") + ", " : ""
+                }${inputValue}`}
+                label={label}
+                placeholder={placeholder}
+                onChange={text => {
+                    const newText = text.replace(
+                        new RegExp(`^${selectedToggleOptions.join(", ")},?\\s?`),
+                        "",
+                    );
+                    setInputValue(newText);
+                    if (onChange) onChange(newText);
+                    setIsOptionsVisible(true);
+                }}
+                onFocus={handleMenuOpen}
+                onBlur={() => setTimeout(() => setIsOptionsVisible(false), 150)}
+                rightAdornments={
+                    <View style={styles.adornmentContainer}>
+                        {inputValue || selectedToggleOptions.length > 0 ? (
+                            <IconButton
+                                name="close"
+                                variant="ghost"
+                                iconSize={18}
+                                onPress={inputValue ? handleClearSingleText : handleClearMultiText}
+                                style={styles.closeIcon}
+                            />
+                        ) : null}
+                        <IconButton
+                            name={isOptionsVisible ? "menu-up" : "menu-down"}
+                            variant="ghost"
+                            iconSize={18}
+                            onPress={handleMenuOpen}
+                            style={styles.menuIcon}
+                        />
+                    </View>
+                }
+            />
+            {isOptionsVisible && (
+                <Menu
+                    anchorEl={inputRef}
+                    open={isOptionsVisible}
+                    onClose={handleMenuClose}
+                    style={styles.menuContainer}
+                >
+                    <ScrollView keyboardShouldPersistTaps="always">
+                        {filteredOptions.map(option =>
+                            !multiple
+                                ? renderSingleSelectItem(option)
+                                : renderMultiSelectItem(
+                                      option,
+                                      selectedToggleOptions.includes(option),
+                                  ),
+                        )}
+                    </ScrollView>
+                </Menu>
+            )}
+        </View>
+    );
+};
+
```

Comment:
Add a Autocomplete.displayName = "Autocomplete" under the component definition. See the other components in the library for more info.